### ISSUE TITLE
License file clarifications

### ISSUE THREAD

**Issue by jengelh** [gcompris/GCompris-voices#3]
The new https://github.com/gcompris/GCompris-voices/blob/master/LICENSE file recently added leaves some open points:
* en/words: is marked as CC-BY-SA, though the website art4apps.org (currently) says CC-BY-NC-SA
* es/words: smart4kids.com appears to have the same design as art4apps, probably NC-SA too? (but doesn't say so)
* af/alphabet & others: is it known which version of GPL/FDL?

**Comment by bdoin** [comment]
Good point, I contact art4apps.org. I can't believe I missed that, probably a change on their site. By the way I participated a little bit in the funding of the art4apps images.

For GCompris contributions, it is GPL V3+. For the FDL version I don't think it is appropriate for audio voices. We can consider them as FDL 1.3+ because we always use the + in our GPL licences.
